docs/reference/remap/flatten.cue

```diff
@@ -0,0 +1,64 @@
+package metadata
+
+remap: functions: flatten: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The array or map to flatten."
+			required:    true
+			type: ["array", "map"]
+		},
+	]
+	return: ["array", "map"]
+	category: "text"
```

Comment:
Is this correct?

Reply:
No. I'm not 100% happy with the categories I have here. Any suggestions for better categories would be welcome!